main/webapp/modules/core/langs/translation-en.json

```diff
@@ -300,6 +300,11 @@
     "core-dialogs/for-null-cell-value-to-empty-str-label": "Convert null value to NULL in INSERT",
     "core-dialogs/choose-export-destination": "Please choose the destination for project export",
     "core-dialogs/export-to-local": "OpenRefine project archive to file",
+    "core-facets/column-tooltip": "Column: $1",
+    "core-facets/expr-tooltip": "Expression: $1",
+    "core-facets/source-tooltip": "Source: $1",
+    "core-facets/facet-current-title": "The current name of the facet is: $1.\n Enter the new facet's name",
```

Comment:
I would perhaps go for something simpler, without giving the current name, just like when editing the project name. Something like `Enter the new facet name`?

Reply:
@wetneb : I respectfully think that this additional information is worth.

When making the decision to change a facet name, the old name might be important.
Also, one side advantage of doing this is making the dialog bigger. Since it can appear far from the facet, bigger maker is more easier to see for the user.

Anyway, if I understand $i18n correctly, changing this string will not break the code, so this is very easy to change in the future.

Regards, A.

Reply:
Could you have the best of both words by having the shorter label but have the old name pre-filled in the dialog?  Maybe also selected to make deletion easier.

Reply:
> Could you have the best of both words by having the shorter label but have the old name pre-filled in the dialog? Maybe also selected to make deletion easier.

Sure, « Current facet name: XYZ » could do. But is it really better?
For the dialog, no luck, it's a JS prompt, and we have little control over that.

Regards,
   Antoine

Reply:
No, I actually meant having the label “Enter facet name:” but putting the old name in the text entry area, highlighted to make it clear it’s changing.  It should be obvious that that is the old name and it makes it easier to modify if you only need to add a suffix, say.

Reply:
Or maybe just “Rename facet $1:” as a label.

Reply:
@gitonthescene : hey, why not… At this point, I guess it's a matter of preference. 

I did your last suggestion, but with « ' » to delimit the name.
I now it's pretty short! ;-)

Regards,
   Antoine